contracts/interfaces/IVault.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Rari Vault interface.
```

Comment:
question: what is Rari?

FYI - The under-development Vault signatures (treasury project) for the ETHVault is pretty close to this one 
https://github.com/windranger-io/windranger-treasury/tree/main/contracts/treasury

Reply:
Since the initial comment, I've caught up on the Rari vaults & their usage in the rewards 🙂